engine/include/cubos/engine/collisions/colliders/simplex.hpp

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include <cubos/core/geom/simplex.hpp>
+
+namespace cubos::engine
+{
+    struct [[cubos::component("cubos/simplex", VecStorage)]] SimplexCollider
+    {
+        glm::vec3 offset;
+        cubos::core::geom::Simplex shape;
+        float margin;
```

Comment:
This should be well documented.
Also, are we using floats or doubles?

Reply:
I don't think we need double precision for collisions. Here speed is a requirement more important than accuracy.